### PR TITLE
got files to import using new query

### PR DESCRIPTION
**Pull request by mak001**

### CODE REVIEW

File: src/Model/ShopifyFile.php [hunk 1]
```diff
@@ -162,24 +252,118 @@ public static function getByShopifyID($shopifyId)
...
+
+    /**
+     * @param int $width
+     * @param int $height
+     * @param string $format
+     * @return ShopifyFileSource|null
+     */
+    public function getTransform($width, $height, $format = null)
+    {
+        if ($this->Type === 'EXTERNAL_VIDEO') {
```
Comment (muskie9) [inline src/Model/ShopifyFile.php:311]: Just noting it on this one, but do we want to do things like `const EXTERNAL_VIDEO = 'EXTERNAL_VIDEO'`, then call `static::EXTERNAL_VIDEO`? Versioned does something similar likely to reduce the possibility of typos downstream if others extend the functionality.

**Review by muskie9 — APPROVED** [review]
Just the one comment which I don't think is a blocker, just figured something we can either look at in this before merge, or after at some point.

**Review by jsirish — COMMENTED** [review]
Import task seems to work well! Old images were deleted as well it seems.

Images seem to show up as archived in the CMS however. Otherwise it works well as far as I can tell

Screen Shot 2021-04-14 at 10 46 25 AM

**Comment by mak001** [comment]
> Import task seems to work well! Old images were deleted as well it seems.
> 
> Images seem to show up as archived in the CMS however. Otherwise it works well as far as I can tell
> 
> Screen Shot 2021-04-14 at 10 46 25 AM

i was unable to replicate the archived status when running the task.  
Screenshot from 2021-04-14 11-34-46

**Review by jsirish — APPROVED** [review]
images seem to be working. they're all draft on my local but that could be due to it being existing data from the old importer

**Pull request merged**